data/base-hunting.yaml

```diff
@@ -1,81 +1,88 @@
 ---
-#   Hunting Zones should be:
-#     1. Added to the correct Province: Zoluren, Ilithi, Therengia‎, Forfedhdar‎, and Qi'Reshalia
-#     2. Ordered decending by min rank
-#   If a Zone already exists in another Province, name it with a prefix denoting this by either an adjective such as "goblins_shaman" instead of "goblins";
-#    if not possible, the abbreviated province in which they are located, i.e., forf_goblins. 
-#      * More detail is better. Grandfathered names are being kept.
-  
-#   Each Hunting Zone should use the following commented format:
-#     - epedia URL followed by 3 spaces a rank range, if unknown, use a ?
-#       * i.e., https://elanthipedia.play.net/Grave_worm   90-130?
-#     - Additional Info
-#       * This is info that is needed such as adding 'tooth' to your loot_additions because black_apes drop teeth which sell like gems.
-#         Another example is adding to Xala_3 "Tactics attacks which steal eyes, give bleeders.", etc.
-#     - Escort Zones should list a province
-
+###################################################################################################
+#                                                                                                 #
+# When adding new Hunting Zones, they should be:                                                  #
+#   1. Added to the correct Province: Zoluren, Ilithi, Therengia‎, Forfedhdar‎, and Qi'Reshalia     #
+#   2. Ordered decending by min rank                                                              #
+# If a Zone already exists in another Province, name it with a prefix denoting this by either     #
+#  an adjective such as "goblins_shaman" instead of "goblins"; if not possible, the abbreviated   #
+#  province in which they are located, i.e., forf_goblins.                                        #
+#   * More detail is better. Grandfathered names are being kept.                                  #
+#                                                                                                 #
+# Each Hunting Zone should use the following commented format:                                    #
+#   - epedia URL followed by 3 spaces and a rank range, if unknown, use a ?                       #
```

Comment:
is this still "3 spaces"?

Reply:
Yeah, just multiplied a few times.

Reply:
Do you think the hooray here looks like a taco?